website/content/api-docs/connect/intentions.mdx

```diff
@@ -450,6 +458,8 @@ The table below shows this endpoint's support for
   for more details.
 </p>
 
+-> The corresponding CLI command is [`consul intention create`](https://www.consul.io/commands/intention/get)
```

Comment:
Copy-paste error
```suggestion
-> The corresponding CLI command is [`consul intention get`](https://www.consul.io/commands/intention/get)
```

Reply:
Actually... for some reason, `consul intention list` is mentioned on the [intentions overview page](https://www.consul.io/commands/intention#basic-examples), but there doesn't appear to be a CLI command page entry for it?

Can you check whether `consul intention list` is a valid command? (If not, let me know, I just won't have time to check it for a few days.) We should probably add that command. And probably reference is from here. I don't actually think `consul intention get` is what we want here... it doesn't seem to list intentions.

Reply:
@jkirschner-hashicorp So `consul intention list` is in fact a valid command but we have no corresponding CLI page

Reply:
I'll add a Github issue for that